mbf_abstract_nav/src/controller_action.cpp

```diff
@@ -115,16 +124,6 @@ void ControllerAction::runImpl(GoalHandle &goal_handle, AbstractControllerExecut
 
   ros::NodeHandle private_nh("~");
 
-  double oscillation_timeout_tmp;
-  private_nh.param("oscillation_timeout", oscillation_timeout_tmp, 0.0);
-  ros::Duration oscillation_timeout(oscillation_timeout_tmp);
-
-  double oscillation_distance;
-  private_nh.param("oscillation_distance", oscillation_distance, 0.03);
```

Comment:
There is a slight change of behavior -- before the param was defaulting to `0.03`, but now it defaults to `0.5`.
`0.5` seems like a better value so I think is ok though